src/test/common/ios/plistBuddy.test.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {PlistBuddy} from "../../../common/ios/plistBuddy";
+
+import * as Q from "q";
+
+suite("plistBuddy", function() {
+    suite("commonContext", function() {
+        test("Should attempt to set plist properties correctly", function() {

```

Comment:
I'd try to explain what correctly means in the name like:
test("setPlistProperty should set the property (and fail), and then add it when the property doesn't exist ")